changelogs/current.yaml

```diff
@@ -4,6 +4,18 @@ behavior_changes:
 - area: tls-inspector
   change: |
     the listener filter tls inspector's stats ``connection_closed`` and ``read_error`` are removed. The new stats are introduced for listener, ``downstream_peek_remote_close`` and ``read_error`` :ref:`listener stats <config_listener_stats>`.
+- area: listener
+  change: |
+    Fixed metric tag extraction so that :ref:stat_prefix <envoy_v3_api_field_config.listener.v3.Listener.stat_prefix>
+    is properly extracted. This changes the Prometheus name from
+    envoy_listener_myprefix_downstream_cx_overflow{} to envoy_listener_downstream_cx_overflow{envoy_listener_address="myprefix"}.
+    This does not affect the Prometheus name if stat_prefix is not set.
+- area: thrift_proxy
+  change: |
+    Fixed metric tag extraction so that :ref:`stat_prefix <envoy_v3_api_field_extensions.filters.network.thrift_proxy.v3.ThriftProxy.stat_prefix>`
+    is properly extracted. This changes the Prometheus name from
+    envoy_thrift_myprefix_request{} to envoy_thrift_request{envoy_thrift_prefix="myprefix"}.
+    This does not affect the Prometheus name if stat_prefix is not set.
```

Comment:
`stat_prefix` is required for ThriftProxy. I think this last sentence should be deleted.

Reply:
interesting did not expect that 27acbd3d9b93ac79e7abdccdb2de9b009fe9b64e